packages/dev/core/src/Meshes/mesh.ts

```diff
@@ -3912,6 +3912,14 @@ export class Mesh extends AbstractMesh implements IGetSetVerticesData {
         throw _WarnImport("GreasedLineMesh");
     };
 
+    /**
+     * @internal
+     */
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars

```

Comment:
You can prefix an unused parameter with `_` instead of adding this eslint line, if you want.

Reply:
I was following the existing code and every Parser has this eslint rule present.

For example:
```javascript
    /**
     * @internal
     */
    // eslint-disable-next-line @typescript-eslint/no-unused-vars
    public static _GroundMeshParser = (parsedMesh: any, scene: Scene): Mesh => {
        throw _WarnImport("GroundMesh");
    };

    /**
     * @internal
     */
    // eslint-disable-next-line @typescript-eslint/no-unused-vars
    public static _GoldbergMeshParser = (parsedMesh: any, scene: Scene): GoldbergMesh => {
        throw _WarnImport("GoldbergMesh");
    };
```